akka-actor-typed-tests/src/test/java/akka/actor/typed/ManualTimerTest.java

```diff
@@ -14,14 +14,14 @@
 import org.junit.Test;
 
 import akka.testkit.typed.TestKit;
-import akka.testkit.typed.ExplicitlyTriggeredScheduler;
+import akka.testkit.typed.javadsl.ExplicitlyTriggeredScheduler;
 import akka.testkit.typed.javadsl.TestProbe;
 
 public class ManualTimerTest extends TestKit {
   ExplicitlyTriggeredScheduler scheduler;
 
   public ManualTimerTest() {
-    super(parseString("akka.scheduler.implementation = \"akka.testkit.typed.ExplicitlyTriggeredScheduler\""));
+    super(parseString("akka.scheduler.implementation = \"akka.testkit.typed.javadsl.ExplicitlyTriggeredScheduler\""));
```

Comment:
Should we have one implementation that implements both the javadsl and scaladsl interfaces here? kind of weird to refer to java/scala from the configuration.

Reply:
I have more changes around this in the other PR, so let's discuss there.